### PR TITLE
Replaced hard coded values of icon size, font, padding and margin with respective variables(#619)

### LINKED ISSUE #619: Replace hard coded values of icon size, font, padding and margin with respective variables
## Current behavior
Values of icon size, font, padding and margin are written with hard coded values

## Expected behavior
Values should be in predefined variables

## Can you handle fixing this bug by yourself?

- [X] YES
- [ ] NO

## Environment details

- Browser: 
- OS: 
- Code Version:

### PR DESCRIPTION
**Pull request by azkyakhan**
### Related Issues
#619 

Closes #619 

### Short Description and Why It's Useful
Replaced hard coded values of icon size, font, padding and margin with respective variables

### Screenshots of Visual Changes before/after (If There Are Any)
NA

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [X] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: components/atoms/a-promo-code.vue [hunk 1]
```diff
@@ -78,7 +78,7 @@ export default {
   }
   &__button {
     --button-height: 2rem;
-    --button-font-size: 0.6875rem;
+    --button-font-size: var(--font-size--xs);
```
Comment (dt2patel) [inline components/atoms/a-promo-code.vue:81]: this is a variable form SFUI 10.x we are currently running SFUI 7. Please update this and any other place you may have used a variable from the new version.

File: components/organisms/o-order-confirmation.vue [hunk 2]
```diff
@@ -230,15 +230,15 @@ export default {
   }
 }
 .paragraph {
-  line-height: 1.875rem;
+  line-height: 1.8;
```
Comment (dt2patel) [inline components/organisms/o-order-confirmation.vue:233]: why did you choose to change this?

File: components/atoms/a-promo-code.vue (hunk 1 above)
Comment (azkyakhan) [reply components/atoms/a-promo-code.vue:81]: Ok I will update it

File: components/organisms/o-order-confirmation.vue (hunk 2 above)
Comment (azkyakhan) [reply components/organisms/o-order-confirmation.vue:233]: I was trying to convert line height to number format.

**Review by dt2patel — APPROVED** [review]
Good to go

**Pull request merged**